software/o_c_REV/HEM_ClockSetup.ino

```diff
@@ -91,44 +108,91 @@ protected:
     }
 
 private:
-    int cursor; // 0=Source, 1=Tempo, 2=Multiply
+    char cursor; // 0=Play/Stop, 1=Forwarding, 2=External PPQN, 3=Tempo, 4,5=Multiply
```

Comment:
Might be worthwhile to extract this to an enum, it'll make it much easier to understand at a glance the different cursor states

Reply:
Yeah, and there's probably a cursor variable in every other applet that should be an enum... I'll work on this.